### DISCUSSION TITLE
Allow using image with http request

### DISCUSSION THREAD

**Discussion by taimaiduc** [streamdeck-linux-gui/streamdeck-linux-gui#190] [Ideas / Feature Requests]
Can we set up a select image via HTTP request? Then we can set dynamic content for image
We can refresh image 1s

Example: Live weather, Live stock value

If we can update this feature it will be a huge impact to setup dynamic content

Stock information 
Weather information 
Uptime server information 
Disk used server information 
Web developments 

![image](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/assets/10907980/1ab831ec-3181-47f8-b4ad-0afbf76916fd)

**Comment by oparada1988** [comment]
a nice workaround for widget
